resources/views/pages/design.twig

```diff
@@ -427,7 +427,7 @@
         {{ bar_chart | raw }}
     </div>
     </div>
-    <script>
+    <script type="text/javascript">
```

Comment:
From [MDN recommendations](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/script/type):

> Authors are encouraged to omit the attribute if the script refers to JavaScript code rather than specify a MIME type.

I would discard this change